web/styles/portico/markdown.css

```diff
@@ -363,21 +367,28 @@
     }
 
     code {
-        /* Copied from rendered_markdown.css */
+        /* Same font-family as zulip.css */
+        font-family: "Source Code Pro", monospace;
+        /* Same as base rules for code elements in rendered_markdown.css */
         font-size: 0.825em;
         unicode-bidi: embed;
         direction: ltr;
+        color: hsl(0, 0%, 0%);
+        border-radius: 3px;
+        /* Different from base rules for code elements in rendered_markdown.css */
         white-space: initial;
```

Comment:
Just noting here that `initial` is not recommended value for properties that can be inherited, see https://developer.mozilla.org/en-US/docs/Web/CSS/initial. It's been this way for a while, so probably not something we need to change, but something to note.